fe/fe-core/src/main/java/com/starrocks/connector/jdbc/JDBCMetadata.java

```diff
@@ -67,13 +67,23 @@ public JDBCMetadata(Map<String, String> properties, String catalogName) {
             LOG.warn("{} not support yet", properties.get(JDBCResource.DRIVER_CLASS));
             throw new StarRocksConnectorException(properties.get(JDBCResource.DRIVER_CLASS) + " not support yet");
         }
+        checkSupportPartitionInformation();
     }
 
     public Connection getConnection() throws SQLException {
         return DriverManager.getConnection(properties.get(JDBCResource.URI),
                 properties.get(JDBCResource.USER), properties.get(JDBCResource.PASSWORD));
     }
 
+    public boolean checkSupportPartitionInformation() {
+        try (Connection connection = getConnection()) {
+            return schemaResolver.setSupportPartitionInformation(
+                    schemaResolver.checkSupportPartitionInformation(connection));
```

Comment:
do you think `schemaResolver.checkAndSetSupportPartitionInformation(connection) ` is more clear?